### ISSUE TITLE
[Content block media-default]: QA: Voiceover does not identify the link text or the label for this link. 

### ISSUE THREAD

**Issue by gopinathan-n** [carbon-design-system/carbon-for-ibm-dotcom#9652]
### Description

Page Element:
Link present at the end of the main region after the H3

Issue Description:
Voiceover does not identify the link text or the label for this link. 
Note: Same issue is found for the link present before the H4 "Tutorials" in "with link list" variant 

Recommend Fix:
Provide A descriptive link text. If it is not possible provide aria-label.

User Impact:
When the screen reader is unable to identify a label, it’s difficult for users to understand the purpose of the control.

Tool:
Voice Over_iPhone

### Component(s) impacted

Content block media-default (Web component canary&staging)

### Browser

Safari

### Carbon for IBM.com version

v1.40.0-rc.0

### Severity

Severity 2 = Aspects of design is broken, and impedes users in a significant way, but there is a way to complete their tasks. Affects major functionality, has a workaround.

### Application/website

Carbon for IBM.com

### Package

@carbon/ibmdotcom-web-components

### CodeSandbox example

https://carbon-design-system.github.io/carbon-for-ibm-dotcom/canary/web-components/?path=/story/components-content-block-media--default

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/carbon-design-system/carbon-for-ibm-dotcom/blob/main/.github/CODE_OF_CONDUCT.md)
- [ ] I checked the [current issues](https://github.com/carbon-design-system/carbon-for-ibm-dotcom/issues) for duplicate issues
